miner/Miner.hs

```diff
@@ -215,6 +236,7 @@ main :: IO ()
 main = do
     execParser opts >>= \case
         Keys -> genKeys
+        BalanceCheck url _miner -> getBalances url _miner
```

Comment:
Let's avoid starting names with `_` outside of record field names - it can interact poorly with warnings and hole fits.

Reply:
Agreed